lib/modules/datasource/hermit/index.ts

```diff
@@ -0,0 +1,159 @@
+import { logger } from '../../../logger';
+import { cache } from '../../../util/cache/package/decorator';
+import { GithubHttp } from '../../../util/http/github';
+import { regEx } from '../../../util/regex';
+import { streamToString } from '../../../util/streams';
+import { id } from '../../versioning/hermit';
+import { Datasource } from '../datasource';
+import { GithubReleasesDatasource } from '../github-releases';
+import { getApiBaseUrl } from '../github-releases/common';
+import type {
+  GithubRelease,
+  GithubReleaseAsset,
+} from '../github-releases/types';
+import type { GetReleasesConfig, ReleaseResult } from '../types';
+import type { HermitSearchResult } from './types';
+
+/**
+ * Hermit Datasource searches a given package from the specified `hermit-pakcages`
+ * repository. It expects the search manifest to come from an asset `index.json` from
+ * a release named index. Any error fetching, parsing and finding the package from the given
+ * registryUrl will be thrown
+ */
+export class HermitDatasource extends Datasource {
+  static readonly id = 'hermit';
+
+  override readonly customRegistrySupport = true;
+
+  override readonly registryStrategy = 'first';
+
+  override readonly defaultVersioning = id;
+
+  override readonly defaultRegistryUrls = [
+    'https://github.com/cashapp/hermit-packages',
+  ];
+
+  pathRegex: RegExp;
+
+  constructor() {
+    super(HermitDatasource.id);
+    this.http = new GithubHttp(GithubReleasesDatasource.id);
```

Comment:
i think this can cause issues, if a user add's ad hostrule for `hermit` instead of `github-releases`. We should do like we do for `pod` datasource
```suggestion
    this.http = new GithubHttp(id);
```

https://github.com/renovatebot/renovate/blob/5fa2aa2f2c0de8eccba46acfde84934a239c4b4b/lib/constants/platforms.ts#L11-L17
https://github.com/renovatebot/renovate/blob/5fa2aa2f2c0de8eccba46acfde84934a239c4b4b/lib/constants/platform.spec.ts#L45